coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -60,10 +61,6 @@ def parse(self):
         elif self.language == "java" and self.docstyle == "default":
             return self._parse_documentation_with_symbols(
                 ("@param  ", " "), "@return ")
-        else:
-            raise NotImplementedError(
-                "Documentation parsing for {0.language!r} in {0.docstyle!r}"
-                " has not been implemented yet".format(self))
```

Comment:
hm you don't use the values from the `docstyle_definition` yet, I would keep this until you change this in the next commits :+1: